src/test/scala/uk/gov/hmrc/play/auth/microservice/filters/AuthorisationFilterSpec.scala

```diff
@@ -185,26 +185,24 @@ class AuthorisationFilterSpec extends WordSpecLike with MatchersResults with Sca
 
     val testAuthConnector = new TestAuthConnector
 
-    class TestAuthorisationFilter(accountProperty: Map[String, String] = Map.empty) extends AuthorisationFilter {
+    class TestAuthorisationFilter(config: Map[String, String] = Map.empty) extends AuthorisationFilter {
       override def controllerNeedsAuth(controllerName: String) = true
 
       private val configMap = Map(
         "DelegateAuthController.authParams.agentRole" -> "admin",
         "DelegateAuthController.authParams.delegatedAuthRule" -> "lp-paye",
-        "DelegateAuthController.authParams.levelOfAssurance" -> levelOfAssurance.toString
+        "DelegateAuthController.authParams.confidenceLevel" -> ConfidenceLevel.toString
       )
 
       override val authParamsConfig =
         new AuthParamsControllerConfig {
-          override def controllerConfigs: Config = ConfigFactory.parseMap(configMap ++ accountProperty)
+          override def controllerConfigs: Config = ConfigFactory.parseMap(configMap ++ config)
         }
 
       override lazy val authConnector = testAuthConnector
-
-
     }
 
-    val authFilter = new TestAuthorisationFilter
+    def authFilter() = new TestAuthorisationFilter()
```

Comment:
sorry for the nitpick, but this should remain a val, making many of the changes above (from `authFilter` to `authFilter()`) obsolete.